src/ui/units/workbooks/components/WorkbookActions/WorkbookActions.tsx

```diff
@@ -62,115 +58,103 @@ export const WorkbookActions: React.FC<Props> = ({workbook, refreshWorkbookInfo}
 
     const collectionsAccessEnabled = Utils.isEnabledFeature(Feature.CollectionsAccessEnabled);
 
+    const onApplyCopy = (workbookId: string | undefined) => {
+        if (workbookId) {
+            history.push(`/workbooks/${workbookId}`);
+        }
+    };
+
     const {useAdditionalWorkbookActions} = registry.workbooks.functions.getAll();
-    const additionalActions = useAdditionalWorkbookActions(workbook);
+    const classNameIconAction = b('icon-action');
+
+    const additionalActions = useAdditionalWorkbookActions(workbook, classNameIconAction);
+
+    const dropdownActions = [...additionalActions];
+
+    if (workbook.permissions.move) {
+        dropdownActions.push({
+            action: () => {
+                dispatch(
+                    openDialog({
+                        id: DIALOG_MOVE_WORKBOOK,
+                        props: {
+                            open: true,
+                            workbookId: workbook.workbookId,
+                            workbookTitle: workbook.title,
+                            initialCollectionId: workbook.collectionId,
+                            onApply: refreshWorkbookInfo,
+                            onClose: () => {
+                                dispatch(closeDialog());
+                            },
+                        },
+                    }),
+                );
+            },
+            text: (
+                <>
+                    <Icon data={ArrowRight} className={classNameIconAction} />
+                    {i18n('action_move')}
+                </>
+            ),
+        });
+
+        if (workbook.permissions.copy) {
+            dropdownActions.push({
+                action: () => {
+                    dispatch(
+                        openDialog({
+                            id: DIALOG_COPY_WORKBOOK,
+                            props: {
+                                open: true,
+                                workbookId: workbook.workbookId,
+                                workbookTitle: workbook.title,
+                                initialCollectionId: workbook.collectionId,
+                                onApply: onApplyCopy,
+                                onClose: () => {
+                                    dispatch(closeDialog());
+                                },
+                            },
+                        }),
+                    );
+                },
+                text: (
+                    <>
```

Comment:
We prefer to use `<React.Fragment>` rather than `<>`

Reply:
fixed